SA/design_concept/content/introduction.typ

```diff
@@ -0,0 +1,7 @@
+#import "../../style.typ": *
+= Introduction

```

Comment:
Do we have mixed line endings? (because of the `^M`)

Reply:
Yes, most definitely. MacOS uses LF, Windows uses CRLF.
I think there is a git configuration to make it consistent

Reply:
I think the ^M things are CRLFs

Reply:
I think we'll need to correct it initially by hand

Reply:
`git add --renormalize .` did the trick. The endings should be all LFs now on the remote and convert to OS default locally.